onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_support_checker.cc

```diff
@@ -1167,6 +1167,50 @@ bool CastOpSupportChecker::IsOpSupportedImpl(const InitializedTensorSet& /* init
 
 #pragma endregion
 
+#pragma region op_depthtospace
+
+class DepthToSpaceOpSupportChecker : public BaseOpSupportChecker {
+ private:
+  bool IsOpSupportedImpl(const InitializedTensorSet& initializers, const NodeUnit& node_unit,
+                         const OpSupportCheckParams& params) const override;
+
+  // DepthToSpace opset 11- has missing key attribute mode, is not supported for now
```

Comment:
>is not supported for now [](http://example.com/codeflow?start=60&length=24)

looks like the opset 11- behavior was DCR. if that's so, would it be easy to support that too?
[https://github.com/microsoft/onnxruntime/blob/0292356bd7ce5b379be8ee9820bfbf4fd321ebc5/onnxruntime/core/providers/cpu/tensor/space_depth_ops.h#L75-L76](https://github.com/microsoft/onnxruntime/blob/0292356bd7ce5b379be8ee9820bfbf4fd321ebc5/onnxruntime/core/providers/cpu/tensor/space_depth_ops.h#L75-L76)